Gemfile

```diff
@@ -52,6 +54,8 @@ group :test do
   gem 'selenium-webdriver'
   # Easy installation and use of web drivers to run system tests with browsers
   gem 'webdrivers'
+  gem 'rails-controller-testing'
```

Comment:
Please tell me way do you need this gem?

Reply:
This gem is needed for controller testing. The gem was required after I ran the tests (that resulted in errors) for the controller for the first time